canonical-web-and-design/kubeflow-news.com
# Add new Twitter section and contact us links

## Done

- Reduce number of blogs per page to 5
- Add new "Follow us on Twitter" card
- Add 2 new contact us links in hero and footer
- Replace node sass with sass

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8035/
- Check page against [brief](https://docs.google.com/document/d/1wzGk9rTyO-7ow_zdF8lpECMkDu1rLiCKq7CHPFwENtA/edit?ts=5ffc500e) and [design](https://github.com/canonical-web-and-design/ubuntu.com/issues/9138)
- Please check contact us links open mail as expected


## Issue / Card

Fixes [#9139](https://github.com/canonical-web-and-design/ubuntu.com/issues/9139)

## Screenshots

![image](https://user-images.githubusercontent.com/58959073/110341404-34eb2080-8022-11eb-86cd-3c6cce02d3ca.png)
